### PR TITLE
Migrate asset types page

### PR DESCRIPTION
**Pull request by alexBinns**
This PR migrates the asset types page.
I have also updated the table to remove the test assets and instead replaced with a flag on for assets that support test. LMK what you think.

### CODE REVIEW

File: src/content/api/asset-types.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: Supported Asset Types
```
Comment (MeganSteenkamp) [inline src/content/api/asset-types.mdx:2]:
```suggestion
title: Asset Types 
```

IMO this looks better in the nav bar ✨

**Comment by MeganSteenkamp** [comment]
@alexBinns I think I prefer the existing table. IMO it looks like you can use `bitcoin.main` as a test asset the way it is laid out now. 
We are also missing `uplinkapi.test` in this table.

**Comment by alexBinns** [comment]
> @alexBinns I think I prefer the existing table. IMO it looks like you can use `bitcoin.main` as a test asset the way it is laid out now. We are also missing `uplinkapi.test` in this table.

We are not gaining anything from having the test and the main separate and I feel it makes the table harder to read

**Comment by alexBinns** [comment]
> @alexBinns I think I prefer the existing table. IMO it looks like you can use `bitcoin.main` as a test asset the way it is laid out now. We are also missing `uplinkapi.test` in this table.

That is what the T is for to indicate what assets can be used with test

File: src/content/api/asset-types.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+title: Asset Types
+description: Asset types supported for payment
+draft: true
+nav:
+  path: API
+  order: 10
```
Comment (MeganSteenkamp) [inline src/content/api/asset-types.mdx:7]: Should this be 12? How are you counting this?
It's the 12 based on the existing docs site.

Comment (alexBinns) [reply src/content/api/asset-types.mdx:7]: Updated

File: src/content/api/asset-types.mdx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+---
+
+The following table describes the Asset Types supported for payments.
+
+Asset types support live and test assets unless specified by supplying the correct suffix e.g. `centrapay.nzd.main` for live and `centrapay.nzd.test` for test.
+
+The Category column refers to the Centrapay asset type representation if
```
Comment (MeganSteenkamp) [inline src/content/api/asset-types.mdx:14]:
```suggestion
The Category column refers to the Centrapay [Asset category](/api/assets/) representation if
```

**Review by MeganSteenkamp — APPROVED** [review]

File: src/content/api/asset-types.mdx (hunk 3 above)
Comment (alexBinns) [reply src/content/api/asset-types.mdx:12]: I have removed this line. It was a previous iteration of the table. I have now updated to have it mentioned in the legend instead.

**Pull request merged**